source/features/more-dropdown.tsx

```diff
@@ -13,15 +13,15 @@ const repoUrl = getRepoURL();
 function createDropdown(): void {
 	// Markup copied from native GHE dropdown
 	appendBefore('.reponav', '[href$="settings"]',
-		<div className="reponav-dropdown js-menu-container">
-			<button type="button" className="btn-link reponav-item js-menu-target" aria-expanded="false" aria-haspopup="true">
-				{'More '}
-				<span className="dropdown-caret"/>
-			</button>
-			<div className="dropdown-menu-content js-menu-content">
-				<div className="dropdown-menu dropdown-menu-se"/>
-			</div>
-		</div>
+		<details className="reponav-dropdown details-overlay details-reset">
+			<summary>
```

Comment:
I think the `btn btn-link` classes can also go onto the `<summary>`. At least this is how i.e. the `Review changes` looks like.

Reply:
Absolutely!